src/adapters/peggedAssets/index.ts

```diff
@@ -91,7 +91,7 @@ import xai from "./xai";
 import iusd from "./iusd";
 import rusd from "./redeemable";
 import ibeur from "./iron-bank-euro";
-import composite from "./composite";
+//import composite from "./composite";
```

Comment:
please uncomment this

Reply:
Commented that to help out with cleanup.
The export of composite has been commented out: https://github.com/DefiLlama/peggedassets-server/blob/d282b345e16f85f2b76823432c1ccfbcda043854/src/adapters/peggedAssets/index.ts#L270

Reply:
Made the change